packages/terra-date-picker/src/react-datepicker/stylesheets/react_datepicker.module.scss

```diff
@@ -193,7 +193,7 @@
     &--selected,
     &--in-selecting-range,
     &--in-range {
-      background-color: $datepicker-selected-color;
+      background-color: darken($datepicker-selected-color, 10%);
```

Comment:
We probably want to check with UX to see if we want to update the background color of the _selected_ date or the _pre-selected_ date.

Reply:
Do we still need this change given the UX feedback on the design input?

Reply:
Was this question ever answered?

Reply:
@saket2403 Is it correct that this change is no longer needed given the new dashed focus style from UX?

Reply:
Yes I have reverted it 76967e7